pkgs/by-name/mi/mixtool/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "mixtool";
+  version = "unstable-2025-07-07";
+
+  src = fetchFromGitHub {
+    owner = "monitoring-mixins";
+    repo = pname;
```

```suggestion
    repo = "mixtool";
```
Never rely on tool and repo matching names. See Catppuccin org for counterexamples

---

Good to know, thank you.